protos/feast/core/Feature.proto

```diff
@@ -42,4 +42,7 @@ message FeatureSpecV2 {
 
     // Metric used for vector similarity search.
     string vector_search_metric = 6;
+
+    // Field indicating the vector length
+    int32 vector_len = 7;
```

Comment:
```suggestion
    int32 vector_length = 7;
```

Reply:
i think the three extra characters are worth it and i'm not sure why we didn't add it to begin with, sorry!